### ISSUE TITLE
Optimize error image is neither stream or buffer when request original file

### ISSUE THREAD

**Issue by ryanhanwu** [jimmynicol/image-resizer#48]
I used to use this library with S3. After upgrading it to 0.10.1, if I request original file, it shows me error as following

```
index-0 (out): [resizer] new stream created!
index-0 (out): [resizer] s3 - 2308ms
index-0 (out): [resizer] identify: no identify
index-0 (out): [resizer] resize - 219ms
index-0 (out): [resizer] filter: none requested
index-0 (out): [resizer] optimize error image is neither stream or buffer
index-0 (out): [resizer] image is neither stream or buffer
index-0 (out): GET /c1315b4d-886f-4099-995a-e15ceb9100c3.png 500 2551.427 ms - -
```

The ***isBuffer*** method of Image return false when requesting original file.
my ```Object.prototype.toString.call(this.contents.parent)``` is '[object undefined]', not sure what it is

**Comment by jimmynicol** [comment]
@ryanhanwu I tried a bunch of different ways to induce this error and was not successful sorry. I also just landed the `v1.0.0` branch which makes a bunch of changes and they may have fixed your particular issue.

**Comment by jimmynicol** [comment]
closing due to in-activity.
